CruCentralCoast/Community/Ministry Teams/MinistryTeamDetailsVC.swift

```diff
@@ -50,7 +50,11 @@ class MinistryTeamDetailsVC: UIViewController {
         
         ministyTitleLabel.text = ministryTitle
```

Comment:
Should probably add "self." on these lines

Reply:
I agree but not sure why this is showing up. It should not be in my pull request